### ISSUE TITLE
[Bug]: Safari/firefox - focus lost when closing popover via ESC

### ISSUE THREAD

**Issue by YuanboXue-Amber** [microsoft/fluentui#29161]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
v9.31.1

macOS - ventura 13.5.2
safari - version 16.6 (18615.3.12.11.2)
firefox - 117.0.1 
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://react.fluentui.dev/?path=/docs/components-popover--default

## Actual Behavior
Use Safari/Firefox. Open docsite, on the nested popover example, open popover via keyboard, focus is on the first button in popover surface.
Then press ESC, popover closes and focus lost to iframe body.

## Expected Behavior
focus should be on popover trigger when using esc to close popover.

### Requested priority

Low

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by ling1726** [comment]
Blocked by microsoft/tabster#305

**Comment by miroslavstastny** [comment]
Duplicate of the https://github.com/microsoft/tabster/issues/305
